jaydebeapi/__init__.py

```diff
@@ -218,7 +218,7 @@ def _jdbc_connect_jpype(jclassname, url, driver_args, jars, libs):
         def _java_array_byte(data):
             return jpype.JArray(jpype.JByte, 1)(data)
     # register driver for DriverManager
-    jpype.JClass(jclassname)
+    jpype.JClass(jclassname)()
```

Comment:
this means instead of just loading the class, we create an instance.  When I did this for SybDriver, it then registered the driver. Without the change, it didn't.